pkgs/misc/uboot/default.nix

```diff
@@ -73,6 +72,9 @@ let
       ]))
       swig
       which # for scripts/dtc-version.sh
+      ncurses # tools/kwboot
+      libuuid # tools/mkeficapsule
+      gnutls # tools/mkeficapsule
```

Comment:
It would probably be even more interesting which tools from those are used. We don't use anything like gnutls-config by chance? I don't think that would be cross friendly necessarily.